pkg/api/v1/pod/util.go

```diff
@@ -51,9 +51,7 @@ func FindPort(pod *v1.Pod, svcPort *v1.ServicePort) (int, error) {
 
 // TODO: remove this function when init containers becomes a stable feature
 func SetInitContainersAndStatuses(pod *v1.Pod) error {
-	var initContainersAnnotation string
-	initContainersAnnotation = pod.Annotations[v1.PodInitContainersAnnotationKey]
-	initContainersAnnotation = pod.Annotations[v1.PodInitContainersBetaAnnotationKey]
+	initContainersAnnotation := pod.Annotations[v1.PodInitContainersBetaAnnotationKey]
```

Comment:
Any reason to follow the logic in `Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec`? this method is called in two places, one is to convey the pod's container and status field, the other one is in the test to construct the pod object, is there any special process that need to take care of?